dockerfiles/hsm-support/keygen.patch

```diff
@@ -0,0 +1,766 @@
+diff --git a/src/eng_back.c b/src/eng_back.c
```

Comment:
Can we maybe add a complete git style patch here with commit message?

Reply:
Done